### DISCUSSION TITLE
Google Chrome Crashes

### DISCUSSION THREAD

**Discussion by codellyoko** [aclap-dev/video-downloadhelper#1118] [Help and Q&A]
### Your question

Hi, I've got VDH for google chrome and it keeps on crashing. Reding forums online it appears the firefox version doesn't experience the same problem so I woyuld like to swap over to firefox. However, my license is for chrome. Is there a way to do this?

I have also had VDH for more than a month.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"D:\\Programme Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"59aaeaf076464e31"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "download_directory": "D:\\Downloads\\VDH",
    "concurrent_downloads_max": 5,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 62
}

**Comment by jcfrog** [comment]
Hi,
I sent you an email.
